processing/src/main/java/org/apache/druid/query/ChainedExecutionQueryRunner.java

```diff
@@ -116,7 +115,7 @@ public ChainedExecutionQueryRunner(
                                 public Iterable<T> call()
                                 {
                                   try {
-                                    Sequence<T> result = input.run(threadSafeQueryPlus, responseContext);
+                                    Sequence<T> result = input.run(queryPlus.withoutQueryMetrics(), responseContext);
```

Comment:
Ok I see, there is at least one reason for doing this, is to not emit queryCpuTime metrics twice, see https://github.com/apache/incubator-druid/pull/1696#discussion_r229729305. It works relying on a crazy combination of side effects, without a single comment. It requires a refactoring.